celery_tasks_prioritizing/README.md

```diff
@@ -0,0 +1,86 @@
+# Prioritizing of the Celery tasks
+
+## Current state
+
+- 2 workers in prod, 1 worker in staging, no priority
+- in each worker we do:
+
+`celery --app="${APP}" worker --loglevel=${LOGLEVEL} --concurrency=1 --prefetch-multiplier=1`
+
+## What do we want to achieve
+
+- process the tasks depending on their priority
+- higher priority: tasks which should provide faster feedback - tasks triggered by user actions,
+  after user pushes/creates PR/comments, he should see that p-s processes this action as soon as possible (sets flags, ...)
+- lower priority: reporting Copr build start, TF results, actions triggered by other events that don't require
+  fast feedback to user
```

Comment:
one nit: could you please turn these into sublists so it's easier to read? e.g.
```
* higher priority:
  * tasks which should provide faster feedback - tasks triggered by user actions
  * ...
```

This is a very good list, I'd love to suggest a slight rewording to a few items (or maybe even interpretation)
* high prio:
  * actionable events to users such as results with logs, final events
  * informations with high value - i.e. first status report
* lower prio:
  * intermediate results
  * long running tasks

Thanks!